docs/chapter4-payload-handoff-format.rst

```diff
@@ -32,6 +32,104 @@ architecture, either 32 or 64 bits.
+                                              This is mainly used by Payloads when initializing page tables. This can eliminate directly SoC accessing from payload for retrieving such information.
+   pci-enum-done     O           Boolean      If this property exists, payload can assume that Platform Init has done full PCI enumeration, resource assignment, and PCI Oprom loading, and those do not have to be done again by payload.
+
+                                              In some payloads, a simple PCI enumeration may still be executed even with “pci-enum-done” to collect PCI device information and bind corresponding drivers to provide services and functionality. (for example, find out network controller and bind network driver to support network functionality)
+   ================= =========== ============ ======================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================
+
+
+4.3 Node: /options/upl-images@<addr> (R)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The behavior of Platform Init when loading a FIT cannot always be known at build
+time. For example:
+
+* Platform Init may select one of several configurations when loading the
+  Payload; the configuration chosen may depend on the hardware that it is
+  running on
+* Platform Init may load images to any address if there is no 'load' property in
+  the image.
+
+The Payload may need to know where one of the images ended up in memory, or
+which configuration was chosen. It cannot find this out by itself. The
+'loaded-image' node provides information to help with these problems.
+
+.. tabularcolumns:: | p{4cm} p{0.75cm} p{4cm} p{6.5cm} |
+.. table:: Node: /options/upl-images@<addr>
+
+   ================= =========== ================== ======================================================================================================================================================================
+   Property          Usage       Value Type         Definition
+   ================= =========== ================== ======================================================================================================================================================================
+   reg               R           prop-encoded array Address and size of FIT image that was loaded and executed to reach this point. This is required if the Payload must have access to the FIT to operate, e.g., if
+                                                    Platform Init does not load the images to a new address. Otherwise, it is optional.
```

Comment:
Its usage is "R" while its description mentioned it is optional for some cases. Is there any conflict?

Reply:
Updated to SD.